### ISSUE TITLE
Spanish translation

### ISSUE THREAD

**Issue by CitoplasmaX** [MidnightBSD/www#1]
I am going to translate this web page. I will make a new directory named "lang" and another named "es" inside with the translated web page.

**Comment by laffer1** [comment]
Sounds good. Try to keep the directory structure underneath /lang/es then same as the root though.

**Comment by CitoplasmaX** [comment]
I had thought of that.

**Comment by CitoplasmaX** [comment]
Could you put a button into webpage to switch the language?

**Comment by laffer1** [comment]
Yes, we can do a button or text links.  In the previous site design, we had links in the side menu. https://web.archive.org/web/20080204223455/http://www.midnightbsd.org:80/

**Comment by CitoplasmaX** [comment]
Fuck. Had MidnightBSD webpage had translator years ago?

**Comment by CitoplasmaX** [comment]
What happened?

**Comment by CitoplasmaX** [comment]
Can i translate the licenses too?

**Comment by laffer1** [comment]
I don't think you want to bother with the license translations. There should be translations online elsewhere we could link to? Not to mention, I'm not sure  the translation would hold up so the english versions would have to be the official license.  

At one point, we had a russian and german translation.  Both of those guys left after awhile and the content got too stale so I had to remove them.

**Comment by CitoplasmaX** [comment]
can i send a pull request with my actual progress?

**Comment by laffer1** [comment]
Sure. On Aug 9, 2017 1:10 PM, CitoplasmaX <notifications@github.com> wrote:can i send a pull request with my actual progress?

**Comment by CitoplasmaX** [comment]
what does background mean when you writes about qt and gtk, front-end?

**Comment by laffer1** [comment]
Which page are you referring to?  

QT and GTK are frameworks for building graphical applications.  For example, I built a GTK 3 app for mport to install packages from X11 rather than having to use the terminal.  (mport-manager in mports)

**Comment by CitoplasmaX** [comment]
<td><span class="medium">Medium</span></td>
					<td>Background in Gtk, QT, etc.</td>

This. is background like front-end?

**Comment by CitoplasmaX** [comment]
https://en.wikipedia.org/wiki/Front_and_back_ends

**Comment by laffer1** [comment]
I need to know the context to tell you what is meant. I doubt background refers to front end but without a link to the text you are referring to, I can't comment further. Normally, if I use background in writing, I'm referring to history or context around something.

**Comment by CitoplasmaX** [comment]
https://github.com/MidnightBSD/www/blob/master/soc/index.html

**Comment by laffer1** [comment]
Oh, in that case background is referring to experience with GTK or QT.

**Comment by laffer1** [comment]
That page is also rather old. It was written when we were still trying to get google to include us in summer of code.

Two of the projects in that list are done:  ZFS and Gjournal.  I'll remove those two later.

**Comment by CitoplasmaX** [comment]
oh.

**Comment by CitoplasmaX** [comment]
but... what are you saying with background?

**Comment by laffer1** [comment]
I merged in what you had so far into http://www.midnightbsd.org/lang/es/ 

it's not linked out yet, but you can see how it looks, etc.

**Comment by CitoplasmaX** [comment]
If there are problems with the links yet to change them, I would prefer to focus only on translating and you would correct the links to as you see fit.

**Comment by CitoplasmaX** [comment]
is it okay?

**Comment by laffer1** [comment]
That's fine.

**Comment by CitoplasmaX** [comment]
When i finish actual translation, can you send me by email new content to translate, then i send you the translation and you add it on the correct page?

**Comment by CitoplasmaX** [comment]
Maybe in December I can continue the translations, the university has caught me very unprepared.
